Readme/integrate-kobiton-teamcity.md

```diff
@@ -0,0 +1,110 @@
+  + [2. Configuring automation test script](#2-configuring-automation-test-script)
+  + [3. Configuring TeamCity project](#3-configuring-teamcity-project)
++ [C. Execution](#c-execution)
+  + [1. Runnig automation test on Kobiton devices](#1-running-automation-test-on-kobiton-devices)
+  + [2. Fetch test session data through Kobiton REST API](#2-fetch-test-session-data-through-kobiton-rest-api)
++ [D. Feedback](#d-feedback)
+## A. PREPARATION
+### 1. Gettting Kobiton username and API key
+In order to execute test on Kobiton, these two must-have parameters must be provided: Username and API key. They are necessary for authenticating with Kobiton.
+
+If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create an account.
+
+Follow the instruction at `IV. Configure Test Script for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/) to get Kobiton username and API key.
+
+### 2. Sample
+In this repository, we have already provided samples for executing automation test with Kobiton through TeamCity:
+  -  [ios-app-test.js](../sample/automation-script/ios-app-test.js): script for execute automation test on Kobiton iOS devices.
+  -  [android-app-test.js](../sample/automation-script/android-app-test.js): script for execute automation test on Kobiton Android devices.
+
+To use the provided sample, follow these steps:
+1. Fork [this repository](https://github.com/kobiton/TeamCity-HockeyApp-Appium-ReactNative).
+2. Create a project in TeamCity to use the forked repository. If you don't know how to do it, please follow [TeamCity document](https://confluence.jetbrains.com/display/TCD9/Creating+and+Editing+Projects#CreatingandEditingProjects-CreateProjectfromURL) for instructions.
+3. Clone the forked repository to configure in the next steps.
+## B. SETUP
+### 1. Getting desired capabilities
+Desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device(s).
+  + Click **Devices** in the top navigation bar.
+
+    ![click devices](./assets/devices.png)
+ 
+  + Hover over any device you want to test with and click on Automation settings button (the one with gear symbol).
+ 
+    ![click auto](./assets/click_auto.png)
+ 
+  + On the left-hand side, you can select your preferred language, App Type, etc.
+  In this example, we use **NodeJS** as the default language of the script. Therefore, choose **NodeJS** in `Language` section and **Hybrid/Native from Apps** in `App type` section. On the right-hand side, Kobiton automatically generates your desired capabilities into code based on the selected options.
+
+  + Write down the generated code as we will be using it in the next step.
+
+    ![click auto](./assets/automation.png) 
+### 2. Configuring automation test script
+In the `automation-script` folder, we will run test for `ApiDemos-debug` application on Android and `UIKitCatalog-Test-Adhoc` apllication on iOS. 
```

Comment:
I think this is a distract information, what is the point of adding this line to the docs